typedb-src/modules/ROOT/pages/quickstart-guide.adoc

```diff
@@ -0,0 +1,534 @@
+= Quickstart guide
+:keywords: getting started, typedb, typeql, tutorial, quickstart, console, studio
+:longTailKeywords: get started with typedb, typedb tutorial, typedb quickstart, learn typedb
+:pageTitle: Quickstart guide
+:summary: Learn how to create a TypeDB database, load schema and data, perform queries.
+:tabs-sync-option:
+:experimental:
+
+This Quickstart guide shows how to set up a TypeDB database and run queries.
+
+== Prerequisites
+
+[#_download_sample_data]
+=== Download schema and data
+
+For this Quickstart guide, you need to download and use the following files:
+
+[#_iam_schema]
+* link:https://github.com/vaticle/typedb-docs/blob/master/typedb-src/modules/ROOT/attachments/iam-schema.tql[iam-schema.tql,window=_blank]
+  -- The TypeQL define query for the IAM schema.
```

Comment:
Do we often use the phrase "define query"?

Reply:
Capitalized Define and Insert as proper names for queries.